docs/src/developer_guide/contributing_to_fuelup.md

```diff
@@ -51,7 +51,7 @@ If you wish to make a significant change that alters current behavior or touches
 This is a rough outline of what a contributor's workflow looks like:
 
 - Make sure what you want to contribute is already tracked as an issue.
-  - We may discuss the problem and solution in the issue.
+  - We may discuss the problem and solution to the issue.
```

Comment:
The original made more sense. In the issue here is referring to a specific github issue where discussion will take place within.

Reply:
done sir